pom.xml

```diff
@@ -826,7 +826,7 @@
             <dependency>
```

Comment:
I think we can remove this dependency now. Created https://github.com/JanusGraph/janusgraph/pull/2319